.github/workflows/ci.yml

```diff
@@ -13,21 +12,19 @@ jobs:
     strategy:
       matrix:
         node-version: [10.x, 12.x, 14.x]
-    services:
-      kafka:
-        image: confluentinc/cp-kafka
-        ports:
-        - 9092:9092
     steps:
-    - uses: actions/checkout@v2
```

Comment:
checkout needs to be left in or else the workflow won't be able to access the repository or its contents